src/app/containers/LinkedData/getAboutTagsContent.js

```diff
@@ -26,7 +26,7 @@ const checkType = types => {
 };
 
 const checkSameAs = uris => {
-  const sameAs = uris.filter(uri => uri.includes('http://dbpedia.org'));
+  const sameAs = uris.filter(uri => uri.startsWith('http://dbpedia.org'));
```

Comment:
Nice work, just one comment.

Github seems to have integrated LGTM and is still complaining about "Incomplete URL substring sanitization".

Reply:
Is this error warning that someone could deploy a service to: http//dbpedia.org.nastypersonurl.com and then they could pass validation using logic like this? As it happens this logic isn't used for this sort of validation so there's no risk here to my knowledge.

Does it just need a slash adding on the end: `'http://dbpedia.org/'`?

Reply:
Adding a trailing slash may fix the error.